.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "googletest"]
+	path = googletest
+	url = git@github.com:google/googletest.git
```

Comment:
Please change url to https://github.com/google/googletest.git

Reply:
Done